drud/ddev
# Remove max_allowed_packet override, fixes #894

## The Problem/Issue/Bug:

A long-ago feature of the mariadb_local container had an override of max_allowed_packet on the command line when running the container. In the meantime we've allowed custom config, and OP #894 points out that it's impossible to override max_allowed_packet.

## How this PR Solves The Problem:

* Remove the command line override.
* Update default max-allowed-packet to better 256M value per discussion in https://github.com/drud/ddev/issues/894#issuecomment-393286945

## Manual Testing Instructions:

* This is pushed as drud/mariadb-local:20180530_remove_max_allowed_packet
* Use `dbimage: drud/mariadb-local:20180530_remove_max_allowed_packet` in a project's config.yaml to use it. 
* Verify that max_allowed_packet defaults to 256M
* Override the max_allowed_packet as described in #894 and verify that behavior is now correct.

## Related Issue Link(s):

OP #894 

## Release/Deployment notes:

This needs to be added to the version.go before release, so a commit needs to be added updating version.go

This is probably OK to add go v0.19.0, but OTOH nobody has discovered it until Jeff, so there's probably no pressure.
